src/Forms/ImportPreviewForm/PackagePreview/PackagePreview.jsx

```diff
@@ -30,11 +30,23 @@ const PackagePreview = ({ onSubmitCallback, importedData }) => {
 
   const languages = useSelector((state) => state.language.languages);
 
+  const [maxFileSize, setMaxFileSize] = useState("");
+
   const [languagePackage, setLanguagePackage] = useState(importedData);
   const [importQueryStatus, setImportQueryStatus] = useState(false);
   const [activateVocabs, setActivateVocabs] = useState(false);
   const [vocabsActive, setVocabsActive] = useState(true);
 
+  useEffect(() => {
+    // fetch max file upload size
+    getInfo()
```

Comment:
I'm not quite sure, but isn't this already saved in redux?

Reply:
Oh you are right. I looked through the code but couldn't find anything, so I was making a specific call. It is stored at login